.scripts/upload_sig_storage_driver.sh

```diff
@@ -1,3 +1,5 @@
+# shellcheck disable=all
```

Comment:
:1st_place_medal:

Reply:
I honestly have no idea why the file still exists. I guess it serves as a warning not to script it?